src/Compilers/Core/Desktop/AnalyzerFileReference.cs

```diff
@@ -208,13 +213,12 @@ internal void AddAnalyzers(ImmutableArray<DiagnosticAnalyzer>.Builder builder, s
                 // If there are no analyzers, don't load the assembly at all.
                 if (!analyzerTypeNameMap.ContainsKey(language))
                 {
-                    this.AnalyzerLoadFailed?.Invoke(this, new AnalyzerLoadFailureEventArgs(AnalyzerLoadFailureEventArgs.FailureErrorCode.NoAnalyzers, null, null));
```

Comment:
If we do preserve this error, I think it should be an info. There may be cases where this info is useful as Manish points out above - but it certainly doesn't deserve warning / error status. 

Could we also hide dependencies passed as analyzer references in the solution explorer? I think we currently show a non-expandable node in the tree view for such dependencies - it would be nice to hide them entirely especially if we are going to have all dependencies including indirect ones specified explicitly.